sdk/metrics/src/test/java/io/opentelemetry/sdk/metrics/SdkMeterTest.java

```diff
@@ -51,12 +51,7 @@ void testLongCounter() {
     assertThat(logs.getEvents()).isEmpty();
 
     sdkMeter.counterBuilder("testLongCounter").build();
-    assertThat(
-            logs.assertContains(
-                    loggingEvent -> loggingEvent.getLevel().equals(WARN),
-                    "Failed to register metric.")
-                .getThrowable())
-        .hasMessageContaining("Metric with same name and different descriptor already created.");
+    logs.assertContains("Found duplicate metric definition");
```

Comment:
A lot of the tests in this code duplicate the test cases in `IdentityTest`, but with less complete validation. I think I should remove the duplicate scenarios to reduce maintenance burden.